lib/beaker/hypervisor/docker.rb

```diff
@@ -264,7 +264,7 @@ def dockerfile_for(host)
 
         # How to start a sshd on port 22.  May be an init for more supervision
         # Ensure that the ssh server can be restarted (done from set_env) and container keeps running
-        cmd = host['docker_cmd'] || ["sh","-c","service #{service_name} start ; tail -f /dev/null"]
+        cmd = host['docker_cmd'] || ["sh","-c","/usr/sbin/sshd; tail -f /dev/null"]
```

Comment:
@alwinmark it looks like with this change, `service_name` isn't being used anymore. Is this true? If that's the case, then the setting statements above should be removed.

If you need to pull out `service_name`, please also refactor the platform-specific statements into a new method, to help get this one under control from a length perspective.